misc/qutebrowser.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 suve <veg@svgames.pl> -->
+<component type="desktop">
+	<id>qutebrowser.desktop</id>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<project_license>GPL-3.0</project_license>
+	<name>qutebrowser</name>
+	<summary>A keyboard-driven web browser</summary>
+	<description>
+		<p>
+			qutebrowser is a keyboard-focused browser with a minimal GUI.
+			It was inspired by other browsers/addons like dwb and Vimperator/Pentadactyl, 
+			and is based on Python and PyQt5.
+		</p>	
+	</description>
+	<launchable type="desktop-id">qutebrowser.desktop</launchable>
+	<screenshots>
+		<screenshot type="default">
+			<image>https://raw.githubusercontent.com/qutebrowser/qutebrowser/master/doc/img/main.png</image>
+		</screenshot>
+		<screenshot>
+			<image>https://raw.githubusercontent.com/qutebrowser/qutebrowser/master/doc/img/downloads.png</image>
+		</screenshot>
+		<screenshot>
+			<image>https://raw.githubusercontent.com/qutebrowser/qutebrowser/master/doc/img/completion.png</image>
+		</screenshot>
+		<screenshot>
+			<image>https://raw.githubusercontent.com/qutebrowser/qutebrowser/master/doc/img/hints.png</image>
+		</screenshot>
+	</screenshots>
+	<url type="homepage">https://www.qutebrowser.org/</url>
```

Comment:
Maybe add [more urls](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url) for bugtracker/faq/help?

Reply:
I've added the bugtracker (pointing at the GitHub issues page). What should the FAQ link point to? The asciidoc, as linked in the readme here?

Reply:
I think the best links would be https://qutebrowser.org/doc/faq.html and https://qutebrowser.org/doc/help/ (for the help).